src/main/scala/mesosphere/marathon/raml/ContainerConversion.scala

```diff
@@ -79,27 +81,57 @@ trait ContainerConversion extends HealthCheckConversion with VolumeConversion wi
       AppCContainer(container.image, container.id, container.labels, container.forcePullImage)
     }
 
-    def create(kind: EngineType, docker: Option[DockerContainer] = None, appc: Option[AppCContainer] = None): Container = {
+    implicit val seccompWrites: Writes[state.Seccomp, Seccomp] = Writes { seccomp =>
+      val unconfined = Some(seccomp.unconfined)
```

Comment:
it is optional in the raml and in mesos protobuf... but it is really strange to me... it opens up things for interpretation that would be best not to.
running a container as normal (historically) is to run `unconfined`
To run `unconfined` just don't define `LinuxInfo` or `Seccomp`.
What does it mean to create a `Seccomp` and not define anything in it... no profile defined and no `unconfined`.   It would be best to not allow this situation... if you want that don't define `seccomp`.   I took the liberty of making `unconfined` not optional in the internal model.   It is either true or false (no guess what the absence means).   If it is absent, then it is false.  Which could be a mistake.   I made the assumption that if `Seccomp` is defined and `unconfined` is not defined then it is likely that `profileName` is defined.   In this case, `unconfined` should be false and will work well.    It doesn't work well under the scenario that `profileName` and `unconfined` is None.    this is most likely written by someone that should NOT be defining seccomp :).    After some thought and this write up.... I will change the raml to enforce that `unconfined` is required in `Seccomp` improving this situation.    ALSO... I dislike negative variable names.   This should have been called `confined`... so the logical is easier for humans to reason about.